src/lua/LuaInput.cpp

```diff
@@ -268,6 +280,8 @@ void LuaInput::Register()
 		{ "SetMouseYInverted", l_input_set_mouse_y_inverted },
 		{ "GetJoystickEnabled", l_input_get_joystick_enabled },
 		{ "SetJoystickEnabled", l_input_set_joystick_enabled },
+		{ "EnableViewTransformationMode", l_input_enable_view_transformation_mode },
```

Comment:
Once #4842 is merged, I'd recommend coalescing these to `SetCapturingInput(bool)`, internally calling `Pi::input.SetCapturingInput()`.

Reply:
I generally deleted these functions from the LuaInput.cpp, because now it works through the SystemView.cpp. See d9ac37975b2f4161bc1350600ca22d3fe5cedf98